### DISCUSSION TITLE
Feature request: compare to baseline

### DISCUSSION THREAD

**Discussion by ICTENCO** [Micke-K/IntuneManagement#233] [Ideas]
Hello,

Thanks for this amazing product. Its great already, but for us the following feature will be amazing:

- Compare settings to a baseline export (this doesn’t work right now since ID is used). So basically we a looking for multitenant comparison.
- It would be even better if we could schedule it / run it as a batch job so we can create a PowerBI dashboard with all the tenants, and show green when the settings match. 

Thank you in advance.

Frank

**Comment by Micke-K** [comment]
Hello,

Thank you!

The first issue is fixed in the  attached file. See [Issue 203](https://github.com/Micke-K/IntuneManagement/issues/203)

I'm planning to include the second one in the next release. Hope to finalise that soon. 

Cheers!

[Compare.zip](https://github.com/Micke-K/IntuneManagement/files/15367385/Compare.zip)
